Jenkinsfile

```diff
@@ -48,7 +53,7 @@ pipeline {
                                         archiveArtifacts 'bin/*.tar.gz'
                                     }
                                 } finally {
-                                    def clean_images = /docker image ls --format "{{.ID}}\t{{.Tag}}" | grep $(git describe --always --dirty) | awk '{print $1}' | xargs docker image rm -f/
+                                    def clean_images = /docker image ls --format "{{.ID}}\t{{.Tag}}" | grep $BUILD_TAG | awk '{print $1}' | xargs docker image rm -f/
```

Comment:
I think this can be simplified because `docker image ls` takes a pattern/glob match:
```
docker image ls --format="{{.ID}}" "$BUILD_TAG" | xargs docker image rm -f
```

(or perhaps `"\*$BUILD_TAG\*"` or something depending what you need to match)

Fine to do this in a followup though

Reply:
👍